notebooks/cookbook/Querying and Modifying Charges and Other Parameters.ipynb

```diff
@@ -136,7 +149,10 @@
     "for i in range(bonded.getNumBonds()):\n",
     "    particle1, particle2, length, k = bonded.getBondParameters(i)\n",
     "    if particle1 == 1 and particle2 == 0:\n",
-    "        bonded.setBondParameters(i, particle1, particle2, length, k_)"
+    "        bonded.setBondParameters(\n",
+    "            i, particle1, particle2, length, \n",
+    "            2666 * unit.kilojoules_per_mole/unit.nanometer**2\n",
+    "        )"
```

Comment:
No need to split this onto four lines.  It makes the code less readable, and it can easily fit on a single line.

Reply:
Sure! If we don't worry about the 80 char limit.

Reply:
If someone wants to view the entry on a VT52 terminal, I admit it will be inconvenient.  But I think we can say [VT100](https://en.wikipedia.org/wiki/VT100) is the earliest terminal we support. :)